scripts/installer/lay-down-os

```diff
@@ -194,6 +194,9 @@ grub2_config "${KERNEL_ARGS}"
 pvgrub_config "${KERNEL_ARGS}"
 install_rancher
 
+echo "__default__:unconfined_u:s0-s0:c0.c1023" > ${BASE_DIR}/etc/selinux/ros/seusers
```

Comment:
I added a check to see if the file exists. I think the problem was just that the file doesn't exist on first install.